### PR TITLE
Restrict tests that use MUMPS

### PR DESCRIPTION
**Pull request by andrsd**
Do not run tests that use MUMPS if it is not available.

### CODE REVIEW

**Comment by loganharbour** [comment]
This doesn't get tested, correct? I thought for a second that 64-bit dof builds with PETSc require disabling mumps, but clearly not on CIVET otherwise next wouldn't be passing without this.

**Comment by fdkong** [comment]
> This doesn't get tested, correct? I thought for a second that 64-bit dof builds with PETSc require disabling mumps, but clearly not on CIVET otherwise next wouldn't be passing without this.

The input file is not actually using mumps: for example (coarse_3D.i), 


```
[Preconditioning]
  active = basic
  [./mumps_is_best_for_parallel_jobs]
    type = SMP
    full = true
    petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
    petsc_options_value = ' lu       mumps'
  [../]
  [./basic]
    type = SMP
    full = true
    petsc_options_iname = '-ksp_type -pc_type -sub_pc_type -sub_pc_factor_shift_type -pc_asm_overlap'
    petsc_options_value = 'gmres      asm      lu           NONZERO                   2             '
  [../]
[]
```

We should enable mumps as well in this PR.

File: modules/porous_flow/examples/flow_through_fractured_media/tests [hunk 1]
```diff
@@ -29,6 +30,7 @@
     abs_zero = 1E-7
     max_parallel = 1 # without the influence of a time derivative, MOOSE believes the porepressure in the ultra-low permeability matrix can be almost anything: the residual are swamped by the residuals in the fracture
     threading = '!pthreads'
+    mumps = True
```
Comment (fdkong) [inline modules/porous_flow/examples/flow_through_fractured_media/tests:33]: If we add this, should we set the input file to use mumps as well?  Do similar things for all other changes.

`Preconditioning/active=mumps_is_best_for_parallel_jobs`

**Review by fdkong — CHANGES_REQUESTED** [review]

**Comment by andrsd** [comment]
> We should enable mumps as well in this PR.

No, that's just me doing search and replace and not checking the actual input file...

**Review by fdkong — APPROVED** [review]

**Pull request merged**